revisionize.php

```diff
@@ -122,21 +123,39 @@ function create_revision($post, $is_original=false) {
     delete_post_meta($post->ID, '_post_original');
   }
 
+	delete_post_meta($new_id,'custom_permalink');  // NEW
+
+
+
+  wp_update_post( array(
+          'ID' => $new_id,
+          'post_name' => ''
+      ));
+
+  //    flush_rewrite_rules();
+
   return $new_id;
 }
 
 function publish($post, $original) {
   if (user_can_publish_revision() || is_cron()) {
     $clone_id = create_revision($original);    // keep a backup copy of the live post.
+	wp_trash_post($clone_id); // NEW
```

Comment:
Why are you trashing the backup?

Reply:
This should actually be an optional thing. But it helps reduce the overall additional documents.